dockerFiles/marklogic-server-centos:base

```diff
@@ -29,6 +29,9 @@ COPY ${ML_RPM} /tmp/marklogic-server.rpm
 RUN yum localinstall -y /tmp/marklogic-server.rpm \
     && rm /tmp/marklogic-server.rpm \
     && yum -y install sudo \
+    && yum -y install epel-release \
+    && yum -y install jq \
```

Comment:
I think we have to update our license NOTICE file if you add this. It looks like it needs updating anyway though.

Reply:
resolving this comment, not using jq in the image file.